pkg/cmd/cli/cmd/set/env.go

```diff
@@ -119,6 +128,9 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 	//overwrite := kcmdutil.GetFlagBool(cmd, "overwrite")
 	resourceVersion := kcmdutil.GetFlagString(cmd, "resource-version")
 	outputFormat := kcmdutil.GetFlagString(cmd, "output")
+	secretName := kcmdutil.GetFlagString(cmd, "from-secret")
+	configMapName := kcmdutil.GetFlagString(cmd, "from-configmap")
+	prefix := kcmdutil.GetFlagString(cmd, "prefix")
```

Comment:
full disclosure: i think i may have been the one who suggested prefix work universally.